pkgs/applications/misc/megasync/default.nix

```
@@ -1,75 +1,111 @@
-{ lib
-, stdenv
-, autoconf
-, automake
-, c-ares
-, cryptopp
-, curl
-, doxygen
-, fetchFromGitHub
-, ffmpeg
-, libmediainfo
-, libraw
-, libsodium
-, libtool
-, libuv
-, libzen
-, lsb-release
-, mkDerivation
-, pkg-config
-, qtbase
-, qttools
-, qtx11extras
-, sqlite
-, swig
-, unzip
-, wget
+{
+  lib,
+  stdenv,
+  c-ares,
+  cmake,
+  cryptopp,
+  curl,
+  fetchFromGitHub,
+  fetchpatch,
+  ffmpeg,
+  hicolor-icon-theme,
+  icu,
+  libmediainfo,
+  libsodium,
+  libtool,
+  libuv,
+  libxcb,
+  libzen,
+  mkDerivation,
+  openssl,
+  pkg-config,
+  qtbase,
+  qtdeclarative,
+  qtgraphicaleffects,
+  qttools,
+  qtquickcontrols,
+  qtquickcontrols2,
+  qtsvg,
+  qtx11extras,
+  readline,
+  sqlite,
+  unzip,
+  wget,
+  zlib,
+  qt5,
 }:
 mkDerivation rec {
   pname = "megasync";
-  version = "4.9.1.0";
+  version = "5.6.1.0";
 
-  src = fetchFromGitHub {
+  src = fetchFromGitHub rec {
     owner = "meganz";
     repo = "MEGAsync";
     rev = "v${version}_Linux";
-    hash = "sha256-Y1nfY5iP64iSCYwzqxbjZAQNHyj4yVbSudSInm+yJzY=";
-    fetchSubmodules = true;
+    hash = "sha256-EjXmx+beYH3J60RidkEbZHFS5ZAOpJKmNC+WCIM84RA=";
```

```suggestion
    hash = "sha256-+022P3W6JXyEJS0vu++nl/SripVJ7LOjUAoCsu5yXhs=";
```
Are you sure your script below is reproducible?